Sources/FusionAudioPlayer/Extensions/Pages/main.page.ext.swift

```diff
@@ -13,8 +13,8 @@ extension MainPageAdapter {
     return self.page?.getWidgetByName("image3") as! SCDWidgetsImage
   }
 
-  var sliderLine1: SCDWidgetsSliderLine {
-    return self.page?.getWidgetByName("sliderLine1") as! SCDWidgetsSliderLine
+  var sldrVolume: SCDWidgetsSliderLine {
```

Comment:
Could you please use full name, i.e. sldr -> slider and other direction.In Apple notation it's more often to read this way "volumeSlider", "customView", etc. etc.